src/hearings/converters/court-location.answer.converter.spec.ts

```diff
@@ -56,7 +56,8 @@ describe('CourtLocationAnswerConverter', () => {
 
   it('should transform type', () => {
     const STATE: State = initialState.hearings;
-    const result$ = listedVenueAnswerConverter.transformAnswer(of(STATE));
+    STATE.hearingRequest.hearingRequestMainModel.hearingResponse.hearingDaySchedule[0].hearingVenueId = '815833';
```

Comment:
const is just a type script keyword, finally this will be var in javascript, it will refer to the reference of the State object, so if it is modified here it will be changed in other place, that's why we need deep clone in other examples.